### PR TITLE
Added small grid with pics on adopt page, edited social media links o…

### PR DESCRIPTION
**Pull request by audiomuse1**
Added small grid with pics on adopt page, edited social media links on donate.css page

### CODE REVIEW

File: adopt.html [hunk 1]
```diff
@@ -92,15 +92,36 @@ <h1 class="display-3">Lorem ipsum dolor sit amet</h1>
...
       </section>
+
       <div class="jumbotron jumbotron-fluid">
         <div class="container">
           <h1 class="display-3">Call To Action</h1>
           <p class="lead">
             This is a modified jumbotron that occupies the
             entire horizontal space of its parent.
           </p>
+          <main class="row text-center">
```
Comment (stuartkilgore) [inline adopt.html]: I don't think nesting a main within another main is best practice. Just make it a div or a section.

File: adopt.html [hunk 2]
```diff
@@ -92,15 +92,36 @@ <h1 class="display-3">Lorem ipsum dolor sit amet</h1>
...
+            <article class="col-md-12 col-lg-4">
+              <img class="img-fluid" class="winebottle" src="https://www.what-dog.net/Images/faces2/scroll0015.jpg" width="209.99px;" height="528.86px">
```
Comment (stuartkilgore) [inline adopt.html]: There isn't a .winebottle class to use.

File: adopt.html [hunk 3]
```diff
@@ -92,15 +92,36 @@ <h1 class="display-3">Lorem ipsum dolor sit amet</h1>
...
+              <br>
+              <a class="btn btn-primary" href="03week\index.html" role="button">Find out more</a>
+            </article>
+
+            <article class="col-md-12 col-lg-4">
+              <img class="img-fluid" class="winebottle" src="https://www.what-dog.net/Images/faces2/scroll0015.jpg" width="209.99px;" height="528.86px">
```
Comment (stuartkilgore) [inline adopt.html]: No .winebottle

File: adopt.html [hunk 4]
```diff
@@ -92,15 +92,36 @@ <h1 class="display-3">Lorem ipsum dolor sit amet</h1>
...
           <h1 class="display-3">Call To Action</h1>
           <p class="lead">
             This is a modified jumbotron that occupies the
             entire horizontal space of its parent.
           </p>
+          <main class="row text-center">
+            <article class="col-md-12 col-lg-4">
+              <img class="img-fluid" class="winebottle" src="https://www.what-dog.net/Images/faces2/scroll0015.jpg" width="209.99px;" height="528.86px">
+              <br>
+              <a class="btn btn-primary" href="03week\index.html" role="button">Find out more</a>
```
Comment (stuartkilgore) [inline adopt.html]: No 03week\index.html to link to

File: adopt.html [hunk 5]
```diff
@@ -92,15 +92,36 @@ <h1 class="display-3">Lorem ipsum dolor sit amet</h1>
...
+            </article>
+
+            <article class="col-md-12 col-lg-4">
+              <img class="img-fluid" class="winebottle" src="https://www.what-dog.net/Images/faces2/scroll0015.jpg" width="209.99px;" height="528.86px">
+              <br>
+              <a class="btn btn-primary" href="03week\index2.html" role="button">Find out more</a>
```
Comment (stuartkilgore) [inline adopt.html]: No 03week\index.html

File: adopt.html [hunk 6]
```diff
@@ -92,15 +92,36 @@ <h1 class="display-3">Lorem ipsum dolor sit amet</h1>
...
+            </article>
+
+            <article class="col-md-12 col-lg-4">
+              <img class="img-fluid" class="winebottle" src="https://www.what-dog.net/Images/faces2/scroll0015.jpg" width="209.99px;" height="528.86px">
```
Comment (stuartkilgore) [inline adopt.html]: No .winebottle

File: adopt.html [hunk 7]
```diff
@@ -92,15 +92,36 @@ <h1 class="display-3">Lorem ipsum dolor sit amet</h1>
...
+              <br>
+              <a class="btn btn-primary" href="04week\index.html" role="button">Find out more</a>
```
Comment (stuartkilgore) [inline adopt.html]: No 04week\index.html

File: css/donate.css [hunk 8]
```diff
@@ -41,3 +41,8 @@
 #amt {
   background-color: #e8faff;
 }
+
+.navbar {
```
Comment (stuartkilgore) [inline css/donate.css]: editing the .navbar class in donate.css will only change the navbar on the donate page. If you meant to edit the navbar for every page, you would need to edit the style.css file instead.
Also, although I think it's more of a stylistic choice, but having a navbar centered would only look good if it was lined up with the rest of the site (e.g. github). If you scroll to the top, you can see how the navbar is kind of centered, but there's some big gutters on the sides of every page. Our site has a lot of full-width images, and I'm not sure having the contents of a navbar be completely centered would look the best.

Comment (audiomuse1) [inline css/donate.css]: Yep exactly, was trying to center the social media links.

File: adopt.html (hunk 1 above)
Comment (audiomuse1) [inline adopt.html]: Good point, I'll change the <main> to <section>

**Pull request merged**